python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1473,6 +1473,15 @@ router bgp {{ router_bgp.as }}
 {%         endif %}
 {%         if vrf.evpn_multicast is arista.avd.defined(true) %}
       evpn multicast
+{%             if vrf.evpn_multicast_gateway_dr_election.algorithm is arista.avd.defined %}
+{%                 if vrf.evpn_multicast_gateway_dr_election.algorithm in ["hrw", "modulus"] %}
```

Comment:
Make this an `else` case instead and move the paths on preference up. It will make it simpler and we don't need to maintain the list of allowed keywords here.

Reply:
Fixed.